.gitignore

```diff
@@ -1,2 +1,3 @@
 vendor
 composer.lock
+/nbproject/private/
```

Comment:
Sure. I didn't notice that my IDE actually added such a thing. I'll remove this ASAP. Sorry for inconvenience.